lib_board_support/api/boards/boards_utils.h

```diff
@@ -4,6 +4,14 @@
 
 #include <xs1.h>
 
+/**
+ * \addtogroup bs_common bs_common
```

Comment:
this group looks a bit off in the generated docs, not a huge deal. It would be nice if the group description (line 10) was included in the pdf, but that would require playing around with the rest options